client/src/pages/Search/AddressSearch.js

```diff
@@ -25,7 +25,8 @@ export default function AddressSearch({ updateFields }) {
     <TextField
       inputRef={googlePlacesRef}
       variant="standard"
-      sx={{ width: "100%", marginBottom: "0.25rem" }}
+      sx={{ width: "60%", minWidth: "230px" }}
+      label="Address"
```

Comment:
Can we make this span 100% on mobile?
![image](https://user-images.githubusercontent.com/51217000/233814473-59aaa227-b817-419b-a4d4-4e150e1bbded.png)